cmd/add.go

```diff
@@ -359,6 +360,7 @@ func init() {
 	addCmd.Flags().BoolVar(&cmdBsubMode, "bsub", false, "enable bsub emulation mode")
 
 	addCmd.Flags().IntVar(&timeoutint, "timeout", 120, "how long (seconds) to wait to get a reply from 'wr manager'")
+	addCmd.Flags().IntVar(&rtimeoutint, "reserve_timeout", 1, "how long (seconds) to wait before a runner exits when there is no more work'")
```

Comment:
I haven't seen how this is used yet, but does it really belong here, and not as an option to a scheduler?

Reply:
It does also make sense a scheduler option. I've been using it on a job by job basis, some need the pod to remain to get files/test timing, others it would just add unnecessary time. 

e.g 

```
# Test 4 + simple commands execute without fail 
echo 'apt-get update && apt-get upgrade -y && apt-get install -y curl' > /tmp/curl.sh
echo {42,24,mice,test} | xargs -n 1  echo echo | /tmp/wr add

# Test we can run configmaps and create files
# reserve_timeout instructs the runner pod to stay alive for long enough to verify the file.
echo 'echo hello world > /tmp/hw' | /tmp/wr add --reserve_timeout 20
```

Where we have a limit to the number of jobs that will run in the cluster before we get timing issues with the tests.

Reply:
If it's just for testing, can you split the tests so you bring up a manager with a scheduler configured with a high reserve_timeout just for the jobs that need it?

It's a bit confusing for normal users to see when adding jobs.

Reply:
I can do, but I also think it might have value. Would it be possible to just set it as a hidden option?